src/ops/cli/parser.py

```diff
@@ -34,6 +34,18 @@ def _get_parser(self):
             help='The cluster config path cluster.yaml')
         parser.add_argument('--root-dir', type=str, help='The root of the resource tree - '
                                                          'it can be an absolute path or relative to the current dir')
+
+        gitfetchHelp = "Whether to call `git fetch` and receive updates from the infrastructure repository." \
+                       " To be used in conjunction with the terraform or helmfile commands"
+        git_fetch_parser = parser.add_mutually_exclusive_group(required=False)
+        git_fetch_parser.add_argument(
+            '--git-fetch', dest='git_fetch', action='store_true', help=gitfetchHelp
+        )
+        git_fetch_parser.add_argument(
+            '--no-git-fetch', dest='git_fetch', action='store_false', help=gitfetchHelp
+        )
+        parser.set_defaults(git_fetch=True)
```

Comment:
Maybe we can keep just one arg and call it more generic like `git_remote` or similar to filter git ops that require interaction with origin

Reply:
How the call will look like?  `ops --git-remote <?>` ?